rpc/wrtc_base_channel.go

```diff
@@ -67,6 +67,15 @@ func newBaseChannel(
 		// to receive messages when the peer is done.
 		ch.cancel()
```

Comment:
I think we can remove this `ch.cancel()`.

Reply:
done